components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/dao/AccessTokenDAOImpl.java

```diff
@@ -1965,9 +1965,11 @@ private Set<AccessTokenDO> getActiveAcessTokenDataByConsumerKey(String consumerK
                     int tenentId = rs.getInt(3);
                     String userDomain = rs.getString(4);
                     String tokenSope = rs.getString(5);
+                    String authorizedOrganizationId = null;
                     String authenticatedIDP = null;
                     if (OAuth2ServiceComponentHolder.isIDPIdColumnEnabled()) {
                         authenticatedIDP = rs.getString(6);
+                        authorizedOrganizationId = rs.getString(8);
```

Comment:
Don't we need to update the query for other query.
`GET_ACTIVE_DETAILS_FOR_CONSUMER_KEY`

But that query seems not executing as there seem to be bug of getting results. There is no 7th column to retrieve at Line 1977. Lets discuss whether that need to be fixed.

Reply:
It has been discussed to remove the IDP column check. https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/2330/files#diff-c7b0374a15c3e8c3009c90d8808546a152e55d667999ba1009c24716ef47c4faR545 -> here also the change has been added only to the new query. hence following the same approach as discussed offline